tests/src/dirs.proj

```diff
@@ -1,5 +1,5 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="dir.props" />
+  <Import Project="Directory.Build.props" />
```

Comment:
This probably should also be an sdk style project.

Reply:
Converting .proj files into sdk style projects does odd things. If we need to do that, it should be a separate change.